src/identifiers.md

```diff
@@ -19,6 +19,12 @@ that does _not_ occur in the set of [keywords].
 > character ranges used to form the more familiar C and Java language-family
 > identifiers.
 
+Links to Unicode descriptions:
```

Comment:
Perhaps it would be nicer to use references like the `[keywords]` and/or `[^non_ascii_idents]` lines below. You wouldn't need this block; the external links would live on the text itself.